<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolbar_rendersettingsgroup</class>
 <widget class="QWidget" name="toolbar_rendersettingsgroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1753</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RenderSettings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="0" column="23">
    <widget class="QCheckBox" name="checkBox_depthLining">
     <property name="text">
      <string>Contours</string>
     </property>
    </widget>
   </item>
   <item row="2" column="22">
    <widget class="QSpinBox" name="spinBox_edgeAwareBlend"/>
   </item>
   <item row="1" column="25">
    <widget class="QSpinBox" name="spinBox_depthLiningSensitivity">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item row="1" column="23">
    <widget class="QLabel" name="label_depthLiningSensitivity">
     <property name="text">
      <string>Sensitivity</string>
     </property>
    </widget>
   </item>
   <item row="1" column="14">
    <widget class="QSpinBox" name="alphaObjectsSpinBox">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>40</number>
     </property>
    </widget>
   </item>
   <item row="2" column="13">
    <widget class="QSlider" name="slider_normals">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="22">
    <widget class="QSpinBox" name="spinBox_edgeAwareDepth"/>
   </item>
   <item row="0" column="4" rowspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="transparencySlider">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="16">
    <widget class="QLabel" name="label_resolution">
     <property name="text">
      <string>Res.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="20">
    <widget class="QSlider" name="slider_edgeAwareRadius">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="13">
    <widget class="QSlider" name="falseColorSlider">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>20</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="22">
    <widget class="QSpinBox" name="spinBox_edgeAwareRadius"/>
   </item>
   <item row="2" column="23">
    <widget class="QLabel" name="label_depthLiningStrength">
     <property name="text">
      <string>Strength</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="19">
    <widget class="QLabel" name="label_mix">
     <property name="text">
      <string>Mix</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_renderMode">
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
   </item>
   <item row="0" column="19">
    <widget class="QLabel" name="label_radius">
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
   </item>
   <item row="1" column="12">
    <widget class="QLabel" name="alphaObjectsLabel">
     <property name="text">
      <string>Objects transp.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="25">
    <widget class="QSpinBox" name="spinBox_depthLiningStrength">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>45</number>
     </property>
    </widget>
   </item>
   <item row="0" column="8" rowspan="5">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_pointSize">
     <property name="text">
      <string>Point size</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="transparencyCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Transp</string>
     </property>
    </widget>
   </item>
   <item row="0" column="14">
    <widget class="QSpinBox" name="falseColorSpinBox">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="2" colspan="2">
    <widget class="QWidget" name="adjust_options" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="QSlider" name="contrastSaturationSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="brightnessLuminanceSpinBox">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="contrastSaturationSpinBox">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="brightnessLuminanceLabel">
        <property name="text">
         <string>Brightness</string>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="brightnessLuminanceSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="contrastSaturationLabel">
        <property name="text">
         <string>Contrast</string>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="20">
    <widget class="QSlider" name="slider_edgeAwareDepth">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="14">
    <widget class="QSpinBox" name="spinBox_normals"/>
   </item>
   <item row="2" column="24">
    <widget class="QSlider" name="slider_depthLiningStrength">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>45</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="7" rowspan="5">
    <widget class="QWidget" name="ramp_options" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QDoubleEdit" name="lineEdit_rampMin">
        <property name="toolTip">
         <string>These values apply to tags, points, spheres, cylinders. The ramps on the boxes are determined by their thickness (Z, blue axis).</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_rampMin">
        <property name="text">
         <string>Ramp Min</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_rampMax">
        <property name="text">
         <string>Ramp Max</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_rampStep">
        <property name="text">
         <string>Ramp Steps</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinBox_rampStep">
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>240</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>240</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleEdit" name="lineEdit_rampMax">
        <property name="toolTip">
         <string>These values apply to tags, points, spheres, cylinders. The ramps on the boxes are determined by their thickness (Z, blue axis).</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox_pointSize">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_renderMode">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="12">
    <widget class="QCheckBox" name="checkBox_normals">
     <property name="toolTip">
      <string>Two types of normals are available: normal type 1 (greyed out), normal type 2 (ticked).</string>
     </property>
     <property name="text">
      <string>Normals</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="15" rowspan="5">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="24">
    <widget class="QCheckBox" name="checkBox_depthLiningStrongMode">
     <property name="text">
      <string>Intense mode</string>
     </property>
    </widget>
   </item>
   <item row="1" column="13">
    <widget class="QSlider" name="alphaObjectsSlider">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>40</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="20">
    <widget class="QSlider" name="slider_edgeAwareBlend">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="24">
    <widget class="QSlider" name="slider_depthLiningSensitivity">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="19">
    <widget class="QLabel" name="label_edgeSensibility">
     <property name="text">
      <string>Edge sens.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="transparencySpinBox">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <widget class="QLabel" name="falseColorLabel">
     <property name="text">
      <string>Hue</string>
     </property>
     <property name="margin">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="pushButton_color">
     <property name="text">
      <string>Selected Color</string>
     </property>
    </widget>
   </item>
   <item row="1" column="17">
    <widget class="QComboBox" name="comboBox_edgeAwareResolution"/>
   </item>
   <item row="0" column="16" colspan="2">
    <widget class="QCheckBox" name="checkBox_edgeAwareBlur">
     <property name="text">
      <string>Smooth render</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qdoubleedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinBox_pointSize</tabstop>
  <tabstop>brightnessLuminanceSlider</tabstop>
  <tabstop>brightnessLuminanceSpinBox</tabstop>
  <tabstop>contrastSaturationSlider</tabstop>
  <tabstop>contrastSaturationSpinBox</tabstop>
  <tabstop>lineEdit_rampMin</tabstop>
  <tabstop>lineEdit_rampMax</tabstop>
  <tabstop>spinBox_rampStep</tabstop>
  <tabstop>falseColorSlider</tabstop>
  <tabstop>falseColorSpinBox</tabstop>
  <tabstop>alphaObjectsSlider</tabstop>
  <tabstop>alphaObjectsSpinBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
